ext/web/lib.rs

```diff
@@ -337,6 +338,15 @@ fn op_encoding_encode_into(
   Ok(())
 }
 
+#[op(v8)]
+fn op_arraybuffer_was_detached<'a>(
+  _scope: &mut v8::HandleScope<'a>,
+  input: serde_v8::Value<'a>,
+) -> Result<bool, AnyError> {
+  let ab = v8::Local::<v8::ArrayBuffer>::try_from(input.v8_value)?;
+  Ok(ab.was_detached())
+}
+
```

Comment:
Why not make this a function in `Deno.core`, like `Deno.core.isProxy`?

Reply:
Like this?

https://github.com/denoland/deno/blob/edaba4d06bd4c218b0217d32197b6e93300c8845/core/01_core.js#L355

Reply:
Yeah. The op would have to be moved to `core/ops_builtin_v8.rs`.